### PR TITLE
Add instructions for backing up and restoring the production database

### PR DESCRIPTION
**Pull request by nb1701**

### CODE REVIEW

File: docs/it-manual/sre-playbook/database-disaster-recovery.md [hunk 1]
```diff
@@ -16,3 +16,75 @@ snapshot. To do this:
...
+
+![pgadmin-clean-before-restore](../assets/pgadmin-clean-before-restore.png)
+
+6. Inspect the tables under `postgres -> Schemas -> Tables` to verify the data was restored properly.
+
+### Option 2: Use the `dumpdb` and `restoredb` commands
+
+This option utilizes the CiviForm Terraform-based deployment system. It will create a temporary EC2 host with access to the database, install the PostgreSQL client, run pg_dump or pg_restore, and then clean up the resources. These commands require the `ssh`, `ssh-keygen`, and `scp` binaries to be available on your system.
+
+{% hint style="danger" %}
```
Comment (dkatzz) [inline docs/it-manual/sre-playbook/database-disaster-recovery.md:75]: **question:** Should this be here too? There's another one a few lines below, so not sure if this does anything?

**Review by dkatzz — APPROVED** [review]
Thanks for making these doc updates! Just a couple of nonblocking comments

File: docs/it-manual/sre-playbook/database-disaster-recovery.md [hunk 2]
```diff
@@ -16,3 +16,75 @@ snapshot. To do this:
...
+
+
+#### Backup
+{% hint style="danger" %}
```
[the remark this answers is not in the document]
Comment (swatkat1) [reply docs/it-manual/sre-playbook/database-disaster-recovery.md:79]: Just a small comment - danger styling is not rendering correctly in rich diff on the PR. Please double check before publishing. 
image

**Review by swatkat1 — APPROVED** [review]

File: docs/it-manual/sre-playbook/database-disaster-recovery.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (swatkat1) [reply docs/it-manual/sre-playbook/database-disaster-recovery.md:79]: ok thanks..

**Pull request merged**